src/main/java/net/minecraftforge/common/command/EntitySelectorManager.java

```diff
@@ -67,7 +67,7 @@ public static void register(String token, IEntitySelectorType type)
      * This method is called in {@link EntitySelectorParser#parse} <br>
      *
      * If the REGISTRY does not contain a custom selector for the command being parsed,
-     * this method returns {@code null} and the vanilla logic in {@link EntitySelectorParser#parseSelector} is used.
+     * this method returns {@code null} and the vanilla logic in {@code EntitySelectorParser#parseSelector()} is used.
```

Comment:
Inconsistency in code formatting, either fully qualified or simple.

Reply:
What do you mean?

Reply:
So, you choose at times to use the fully qualified name or simple names in `@code` tags arbitrarily. The same applies to `@link` tags. It should standardized across all projects on what to do for this. Though, since this is out of scope, it could probably just be an issue for now.